templates/quickstart-confluence-master.template.yaml

```diff
@@ -1721,6 +1721,21 @@ Resources:
       FromPort: -1
       ToPort: -1
       SourceSecurityGroupId: !Ref SecurityGroup
+# Optional: Cloudwatch dashboard to be created when CloudWatch is enabled
+  CloudWatchDashboard:
+    DependsOn:
+      - DB
```

Comment:
What would happen if the `DB` wasn't in the `DependsOn`? Will it create the dashboard anyway or will it fail?

Reply:
I'm not completely sure - I reckon CFn will wait for the DB anyway as the resource is referrred to. Will check and update

Reply:
As expected, CFn is waiting for provisioning completion before going ahead to provision the dashboard